modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -550,4 +553,71 @@ private function fieldFormatSocialElement($items, $body) {
       $body->addChild($social);
     }
   }
+
+  /**
+   * Add ads if configured in settings
+   */
+  public function addAdsFromSettings() {
+    $ad_type = variable_get('fb_instant_articles_ad_type', AdTypes::FB_INSTANT_ARTICLES_AD_TYPE_NONE);
+    if ($ad_type === AdTypes::FB_INSTANT_ARTICLES_AD_TYPE_NONE) {
+      return;
+    }
+    $dimensions = variable_get('fb_instant_articles_ads_dimension', AdTypes::FB_INSTANT_ARTICLES_AD_TYPE_NONE);
```

Comment:
Actually.. It does not make sense in that line at all (my eyes were in the wrong place) 👓 I will fix that.